sechub-scan/src/main/resources/templates/report/html/scanresult.html

```diff
@@ -150,31 +462,63 @@ <h2 class='yellowFindingHeadline'>Yellow findings</h2>
 			<div th:if="!${greenList.isEmpty()}">
 	            <h2 class='greenFindingHeadline'>Green findings</h2>
 				<table class='findingsTable greenFindingsTable'>
```

Comment:
Repeating the same code three times does not add value.

Reply:
done